README.md

```diff
@@ -23,12 +23,20 @@ This is a TCP server that is comptaible with the [Privacy Pass](https://github.c
 
 To run the server:
 
-`go run server/main.go --key testdata/p256-key.pem --comm testdata/test-p256-commitment`
+`go run server/main.go --sign_key testdata/p256-key.pem --comm testdata/test-p256-commitment`
+
+here, `sign_key` is the current secret key used for signing, `comm` is the public commitment to the signing key.
 
 To demo token issuance:
 
 `cat testdata/bl_sig_req | nc localhost 2416`
 
+Keys that are only valid for verifying token redemptions can be provided in a single `.pem` file. For example, using:
+
+`--redeem_keys testdata/p256-redeem-keys.pem`
```

Comment:
This is a little odd to have multiple private keys stacked together in a single pem, but I guess it's fine.